concent_api/core/views.py

```diff
@@ -193,40 +193,30 @@ def receive(_request, _message):
             return decoded_message_data
         return None
 
-    if isinstance(decoded_message_data, message.RejectReportComputedTask):
-        force_report_computed_task = Message.objects.get(
-            type    = message.ForceReportComputedTask.TYPE,
-            task_id = decoded_message_data.cannot_compute_task.task_to_compute.compute_task_def['task_id']
-        )
-        raw_force_report_computed_task = force_report_computed_task.data.tobytes()
+    assert isinstance(decoded_message_data, message.RejectReportComputedTask)
+    #  ReceiveStatus can't have diffrent messages than ForceReportComputedTask, AckReportComputedTask or RejectReportComputedTask
```

Comment:
If you make a comment about something, put it above the line you're commenting on. Otherwise may not be clear what you're referring to.